webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         // instead it expects a global object called 'adapter' for that.
         // Let's make that object available.
         new webpack.ProvidePlugin({
-            adapter: 'webrtc-adapter'
+            adapter: ['webrtc-adapter', 'default']
```

Comment:
Not 100% sure why this change was needed, but it looks like the slight change in dependency tree moved this inside a module and webpack recommends this syntax to go inside of it.